frontend_build/src/webpack.config.base.js

```diff
@@ -12,9 +12,23 @@
  *  and used as a template, with additional plugin-specific
  *  modifications made on top.
  */
+var path = require('path');
+
+/*
+ * This is a filthy hack. Do as I say, not as I do.
+ * Taken from: https://gist.github.com/branneman/8048520#6-the-hack
+ * This forces the NODE_PATH environment variable to include the main
+ * kolibri node_modules folder, so that even plugins being built outside
+ * of the kolibri folder will have access to all installed loaders, etc.
+ * Doing it here, rather than at command invocation, allows us to do this
+ * in a cross platform way, and also to avoid having to prepend it to all
+ * our commands that end up invoking webpack.
```

Comment:
I read this, but I'm not sure I understand it... but I understand that you know what you're doing :)